rocAL_pybind/CMakeLists.txt

```diff
@@ -31,21 +35,17 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR})
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../cmake)
-set(ROCM_PATH /opt/rocm CACHE PATH "Deafult ROCm Installation Path")
 
 set(CMAKE_CXX_STANDARD 17)
 set(BUILD_ROCAL_PYBIND true)
+set(ROCAL_PYTHON ON)
```

Comment:
@kiritigowda this was removed by Hansel, since it was redundant. We use only `BUILD_PYPACKAGE` now. Please keep CMakeLists changes on develop as is